tensorrt_llm/_torch/auto_deploy/custom_ops/quantization/torch_quant.py

```
@@ -587,6 +591,68 @@ def _torch_fake_quant_finegrained_fp8_linear_fake(
     return torch.empty((*input.shape[:-1], out_features), dtype=input.dtype, device=input.device)
 
 
+@torch.library.custom_op(
+    "auto_deploy::torch_fake_quant_deepseek_v4_wo_a_grouped_finegrained_fp8_linear",
+    mutates_args=(),
+)
+def torch_fake_quant_deepseek_v4_wo_a_grouped_finegrained_fp8_linear(
+    input: torch.Tensor,  # [B, S, G, Dg]
```

Ruff (0.15.11)

[error] Function argument `input` is shadowing a Python builtin

(A002)